storage/txcache/txListForSenderAsSortedMapItem.go

```diff
@@ -0,0 +1,81 @@
+package txcache
+
+import (
+	"math"
+
+	"github.com/ElrondNetwork/elrond-go/storage/txcache/maps"
+)
+
+type senderScoreParams struct {
+	count uint64
+	// Size is in bytes
+	size uint64
+	// Fee is in micro ERD
```

Comment:
we should not rely that 1ERD has 10^18 subdivisions

Reply:
According to our discussion, this issue will be addressed in a future task.